test/handle_access_token_response_test.dart

```diff
@@ -47,6 +47,12 @@ void main() {
       expect(() => handleError(headers: {
         'content-type': 'text/plain'
       }), throwsFormatException);
+    }, skip: 'text/plain content-type is accepted now');
```

Comment:
Ah, ok. So I'll remove the `skip`, and make sure that it's still forbidden. Thanks for clarifying.